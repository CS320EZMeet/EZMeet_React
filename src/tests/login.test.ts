import renderer from 'react-test-renderer';
import Login from '../pages/login';

it('renders login correctly', () => {
    const tree = renderer
        .create( Login() )
        .toJSON();
    expect(tree).toMatchSnapshot();
});