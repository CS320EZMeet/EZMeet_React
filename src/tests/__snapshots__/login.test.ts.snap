// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<h1>
  I'm the login page
</h1>
`;

exports[`renders login correctly 1`] = `
<div
  className="col-md-12"
>
  <div
    className="card card-container"
  >
    <form
      action="#"
      onReset={[Function]}
      onSubmit={[Function]}
    >
      <div
        className="form-group"
      >
        <label
          htmlFor="username"
        >
          Username
        </label>
        <input
          className="form-control"
          name="username"
          onBlur={[Function]}
          onChange={[Function]}
          type="text"
          value="username"
        />
      </div>
      <div
        className="form-group"
      >
        <label
          htmlFor="password"
        >
          Password
        </label>
        <input
          className="form-control"
          name="password"
          onBlur={[Function]}
          onChange={[Function]}
          type="password"
          value="password"
        />
      </div>
      <button
        className="btn btn-primary btn-block"
        type="submit"
      >
        Login
      </button>
    </form>
  </div>
</div>
`;
